package.json

```diff
@@ -9,17 +9,17 @@
     "validate-app-config": "ajv validate -s ./node_modules/@alfresco/adf-core/app.config.schema.json -d ./app/.tmp/app.config.json --errors=text --verbose",
     "assemble-app-config": "envsub --env-file ./.env --env APP_CONFIG_ECM_HOST={protocol}//{hostname}{:port} --env APP_CONFIG_PROVIDER=ECM --env APP_CONFIG_AUTH_TYPE=BASIC --env APP_CONFIG_OAUTH2_HOST=http://localhost:4200/auth/realms/alfresco --env APP_CONFIG_OAUTH2_CLIENTID=alfresco --env APP_CONFIG_OAUTH2_IMPLICIT_FLOW=true --env APP_CONFIG_OAUTH2_SILENT_LOGIN=true --env APP_CONFIG_OAUTH2_REDIRECT_SILENT_IFRAME_URI={protocol}//{hostname}{:port}/assets/silent-refresh.html --env APP_CONFIG_OAUTH2_REDIRECT_LOGIN=/ --env APP_CONFIG_OAUTH2_REDIRECT_LOGOUT=/logout --all ./app/src/app.config.json.tpl ./app/.tmp/app.config.json",
     "prestart": "mkdir -p ./app/.tmp && npm run assemble-app-config && npm run validate-app-config",
-    "start": "ng serve",
+    "start": "nx serve",
     "start:prod": "npm run validate-app-config && node --max-old-space-size=8192 node_modules/@angular/cli/bin/ng serve --configuration production --open",
     "prebuild": "mkdir -p ./app/.tmp && cp ./app/src/app.config.json.tpl ./app/.tmp/app.config.json",
-    "build": "ng build content-ce",
+    "build": "nx build content-ce",
     "build.release": "npm run build -- --configuration=production,release",
-    "build-libs": "ng build aca-shared && ng build adf-office-services-ext && ng build aca-about && ng build aca-viewer && ng build aca-preview && ng build aca-folder-rules && ng build aca-content",
-    "test": "ng test",
-    "lint": "NODE_OPTIONS=--max_old_space_size=4096 ng lint",
+    "build-libs": "nx build aca-shared && nx build adf-office-services-ext && nx build aca-about && nx build aca-viewer && nx build aca-preview && nx build aca-folder-rules && nx build aca-content",
```

Comment:
can we investigate how to make project dependencies so that this script can be removed? the ADF should have some examples on how one project causes building other dependant ones

Reply:
ok I will check thanks

Reply:
Looks like it works already in that way by default. After calling: 
nx build content-ce (which is build script in package.json)

it builds also all needed libraries. See screenshot: 
<img width="966" alt="image" src="https://user-images.githubusercontent.com/115619721/233979415-4d054072-d44e-4846-80d9-2af2c289897d.png">

So I just removed that script